instance/github.py

```diff
@@ -86,6 +87,18 @@ def get_settings_from_pr_body(pr_body):
         return ''
 
 
+def get_ephemeral_databases_from_pr_body(pr_body):
+    """
+    Return True if the PR body specified that the sandbox should use ephemeral
+    databases, False if it specifies persistent databases, or None otherwise
+    """
+    if re.search(r'..Sandbox:?...*\(ephemeral databases?\)', pr_body):
+        return True
+    if re.search(r'..Sandbox:?...*\(persistent databases?\)', pr_body):
+        return False
```

Comment:
@omarkhan Also, can you add instructions in the README about this? I know currently the rest of the end-user documentation is missing, so you don't need to integrate the rest of the gist I was giving, but this would still need to be documented somewhere.